xAPI.md

```diff
@@ -509,12 +509,56 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 8.0 Getting More Out of xAPI
 
+xAPI was developed to be as lightweight as possible, but able to be focused to be powerful.  xAPI defines only 
+one verb (for voiding Statements) and only one activity type (cmi.interaction).  Obviously, this isn't meant to 
+be the extent of what xAPI can do.  xAPI is limitless in how far it can be taken, which is why this document 
+doesn't attempt to define a "starting set", because it would encourage adoption of only said set.  Many of the 
+Resources, such as State and Activity Profile, are intentionally left as "a set of name/value" pairs to not try 
+to force a certain practice.  The very notion of IRIs and suggested resolvable metadata be at the location is a 
+way to continuously increase the value of xAPI by not tying definitions down permanently.  
+
```

Comment:
This intro seems to be mainly about the scope of xAPI. I think it dives into details too much: the important point isn't there the spec defines one verb and activity type, but that generally it defines structure and not the content of that structure. This enables it to be flexible and applied in contexts beyond e-learning, including activities that haven't even been invented yet. 

This section should say:
* xAPI defines the structure of communication but not the content
* this is to make it flexible. 
* It's envisaged that communities of practice will define content for their particular use cases
* It's important that they do so